setup.cfg

```diff
@@ -1,3 +1,15 @@
+[isort]
+skip =
+    _vendor
+    __main__.py
+multi_line_output = 5
```

Comment:
This essentially sets the kind of indentation that is used by isort when there are imports that span multiple lines.

5 is the hanging import style that pip currently uses.